examples/launch_chrome.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This can bed it's own MR if we want to reduce line count, it is really useful so imo should go in either way.